apps/landingPage/components/Header.js

```diff
@@ -50,115 +50,108 @@ const logos = [
 ];
 export default function Header() {
   return (
-    <Container>
-      <PlaneImageMobile />
-      <UpperHeader>
-        <Logo src="/static/logo_white.png" alt="kiwi.com logo" />
-      </UpperHeader>
-      <Columns>
-        <Headline>
-          <Heading inverted={true} element="div">
-            <WrappedText>
-              Use <ProductName>Margarita</ProductName> and create your own
-              Kiwi.com for free
-            </WrappedText>
-          </Heading>
-          <Text size="large" element="div" weight="bold" spaceAfter="medium">
-            <HeadlineTextWrapper>
-              We prepared everything you will need — data, web and mobile apps.
-              All is based on modern technologies and it's open source.
-            </HeadlineTextWrapper>
-          </Text>
-          <ButtonWrapper>
-            <ButtonGroup>
-              <Button
-                circled
-                width={140}
-                href="https://kiwicom.github.io/margarita/docs/"
-              >
-                Read more
-              </Button>
-              <ButtonLink
-                type="secondary"
-                href="https://github.com/kiwicom/margarita"
-              >
-                <GithubLogo src="/static/github.png" alt="Github" />
-                <Text type="white" weight="bold">
-                  View on GitHub
-                </Text>
-              </ButtonLink>
-            </ButtonGroup>
-          </ButtonWrapper>
-        </Headline>
-        <PhoneImage src="/static/search.png" alt="Booking" />
-      </Columns>
-      <Logos>
-        {logos.map(logo => (
-          <LogoCard
-            key={logo.title}
-            icon={logo.icon}
-            title={logo.title}
-            text={logo.text}
-          />
-        ))}
-      </Logos>
-    </Container>
+    <>
+      <Container>
+        <Content>
+          <UpperHeader>
+            <Logo src="/static/logo_white.png" alt="kiwi.com logo" />
+          </UpperHeader>
+          <Columns>
+            <Headline>
+              <Motto inverted={true} element="div">
```

Comment:
Why do we need to remove <Headline /> from orbit here?

Reply:
We need different font and different styling